src/shared/main-styles.scss

```diff
@@ -1,47 +1,50 @@
-// $beer-text-color:black;
-// $bg-color:whitesmoke;
-// $navbar-text-color: white;
-// $title-color:black;
-// $navbar-bg-color:gray;
-// $button-positive-color:#4CAF50;
-// $button-previous-page-color:red;
-// $button-change-pictures-color:gray;
-// $button-text-color:white;
-// $input-border-color:green;
-// $input-text-color:black;
+$beer-text-color: black;
+$bg-color: whitesmoke;
+$navbar-text-color: white;
+$title-color: black;
+$navbar-bg-color: gray;
+$button-positive-color: #4caf50;
+$button-previous-page-color: red;
+$button-text-color: white;
+$button-positive-hover: #3e8e41;
+$input-border-color: green;
+$input-text-color: black;
+$delete-beer-button-color: red;
+$beer-text-size: 18px;
+$button-previous-page-color-hover: darkred;
+$description-name-color: maroon;
+$icon-favourite-bg-color: yellowgreen;
 
-// @mixin title {
-//   font-size: 40px;
-//   text-align: center;
-//   letter-spacing: 3px;
-//   font-family: fantasy;
-//   margin-top: 80px;
-//   color: $title-color;
-// }
+@mixin title {
+  font-size: 40px;
+  text-align: center;
+  letter-spacing: 3px;
+  font-family: fantasy;
```

Comment:
font-family should always have some backup fonts described

Reply:
Done